product.json

```diff
@@ -236,6 +244,7 @@
 		{
 			"name": "quarto.quarto",
 			"version": "1.118.0",
+			"sha256sum": "34bd034a7961cba3dbc6f1e2f2eb23d50b29164714351eeb5913847a98e343c9",
```

Comment:
Actually, could we bump Quarto to 1.119.0 in this PR? I just did a new release and it would be great to get that integrated now.

Reply:
Good idea, done!